### PR TITLE
SauceLabs Integration

### PR DESCRIPTION
**Pull request by FinalAngel**
Getting SauceLabs to work

### CODE REVIEW

**Comment by intellectronica** [comment]
:+1: awesome!

**Comment by FinalAngel** [comment]
doesn't work yet @intellectronica ;)

**Comment by intellectronica** [comment]
ah i missed the wip label. i guess ... it's going to be awesome when it works :)

File: tests/unit/test.base.js [hunk 1]
```diff
@@ -14,6 +14,11 @@
     'use strict';
 
     describe('base.js', function () {
+        beforeEach(function () {
+            fixture.setBase('tests/fixtures');
+            this.result = fixture.load('outdatedBrowser.html');
```
Comment (vxsx) [inline tests/unit/test.base.js]: I think `this.fixtures` would be better, `result` is very ambiguous

**Comment by vxsx** [comment]
LGTM

File: tests/unit/test.base.js (hunk 1 above)
Comment (FinalAngel) [inline tests/unit/test.base.js]: there is already a variable called `fixture` using `fixture.setBase('tests/fixtures');` which might be confusing

**Pull request merged**